### ISSUE TITLE
HW version renders garbled mess in special stages

### ISSUE THREAD

**Issue by CrostDaergon** [SaturnSH2x2/Sonic-CD-11-3DS#11]
While the special stages run miles better in the HW version in terms of framerate, they're unplayable because the floor isn't displayed, and the background is made of nothing but a mess of tiles :
![2021-09-07_14-13-09 809_top](https://user-images.githubusercontent.com/71353376/132343014-b6254cc4-83ed-48e7-b317-e2474e69a368.png)
![2021-09-07_14-13-09 809_bot](https://user-images.githubusercontent.com/71353376/132343016-11de3809-e3a4-4738-9470-eb10086e348d.png)
Is this happening to other people too ?

**Comment by lividhen** [comment]
I don't believe they've been implemented yet. That happens for me too.

**Comment by SaturnSH2x2** [comment]
Yeah, this is mainly the result of me messing around trying to figure out how layouts for special stages are loaded. I'll remove the garbled tiles from future releases until the floor layer is properly implemented. And until said floor layer is implemented, I'll leave this issue open.
